doc/Developing-Step-By-Step-Core.html

```diff
@@ -324,14 +325,14 @@ <h3>Creating Person Application Service</h3>
 <p>Application services are located in
 <strong>.Application</strong> project. We create first application service to 
 get people from server. We're creating an <strong>interface</strong> to define 
-the person application service (while this interface is optional, we suggest you 
-to create it):</p>
+the person application service (while this interface is optional, it's suggested you 
```

Comment:
it's suggested you **for** you (...to create it)

Reply:
Can we also answer "Why?" it's suggested for us to use interfaces? I only started using mvc when i picked up ABP. But interfaces isnt something that really makes any sense to me, and i'm still trying get past the why i (we?) have to bother with it. 
If the interface is optional then nothing is lost if it isnt included, then telling us to put it, in with a good reason, helps to encourage positive practices. Otherwise it's just a thing we do, because we did it that one time...